libs/src/sdk/api/playlists/PlaylistsApi.ts

```diff
@@ -384,7 +384,7 @@ export class PlaylistsApi extends GeneratedPlaylistsApi {
     const updatedMetadata = {
       playlistId,
       playlistContents: {
-        trackIds: metadata.playlistContents.map(
+        trackIds: (metadata.playlistContents ?? []).map(
```

Comment:
I think we should null check at the `playlistContents` level, otherwise we could overwrite when metedata.playlistContents is not passed in